apis/supernova/src/index.js

```diff
@@ -1,6 +1,7 @@
 import generator from './generator';
+import JSONPatch from './json-patch';
 
-export { generator };
+export { generator, JSONPatch };
```

Comment:
Feels a little bit weird, the way we import and then export generator and and JSONPatch like this. Maybe we should consider creating some kind of common internal pkg instead for shared stuff, but we don't have to do it right now.

Reply:
it is a bit weird, but supernova is an internal package, so no too far off.